addons/event_booth_sale/views/event_booth_views.xml

```diff
@@ -41,13 +41,13 @@
     </record>
 
     <record id="event_booth_view_tree" model="ir.ui.view">
```

Comment:
Looks like this view is removed in your upgrade script? Not sure why?

Reply:
Yes thanks for noticing it. I think I added it before the refactoring (with the context variable) because the logic was reversed before (the inheritance was reversed). I have removed that deletion.